Server/jest.config.js

```diff
@@ -13,12 +13,12 @@ module.exports = {
         ".mock.ts"
     ],
     collectCoverageFrom: [
-    'src/controllers/*.ts',
-    'src/middleware/*.ts',
-    'src/models/*.ts',
-    'src/services/*.ts'
-  ]
-	,
+        'src/controllers/**/*.ts',
```

Comment:
is this part of the admin story or is this the fix you did for JT?

Reply:
Independent of the admins

Reply:
That reminds me to do this for methods, which is required for this story